spec/models/community_users.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
+
+describe CommunityUsers do
+  let(:student) { create(:student) }
+  let(:student_looking_for_coach) { create(:student, :interested_in_coaching) }
+  let(:coach) { create(:coach, :available) }
+  describe '#all' do
+    before do
+      @params = {
+          role: '',
+          interest: '',
+          availability: '',
+          search: '',
+          location: '',
+          page: 1,
+          sort: '',
+          direction: 'asc'
+        }
+
+      @users = User.ordered(@params[:sort], @params[:direction])
+                   .group('users.id')
+                   .with_all_associations_joined
+
+      @community = CommunityUsers.new(@params, @users)
+    end
+
+    context 'return filtered users' do
```

Comment:
Since there's only one `context` I don't see the point in having it 😉  it just increases the indentation level in this case and does not really give more information.